### PR TITLE
Display Prerequisite Events

### PR DESCRIPTION
**Pull request by jrjohnson**
Refs ilios/frontend#1323
Adds prework section to week at a glance events

WIP:
- [x] Extract ILM prework into the top of the week at a glance
- [x] Sort out what should display when a lot of offerings are present in a prequisite 
- [x] Requires https://github.com/ilios/ilios/pull/2292
- [x] On single event view when and offering is prework it should display the offering date and then on the next line show the due prior to date.
- [x] In single event view the name of the postrequisite should be a link to that single event
- [x] Each event in week at a glance has a prework title and that should be the same size as otehr second level elements.
- [x] When there is no pre-work for a week the section should be removed instead of saying 'none`.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview for *ilios-common* ready!

Built with commit 9bac7e56c56405661ccc32f638e313404f35824e

https://deploy-preview-411--ilios-common.netlify.com

File: addon/components/ilios-calendar-week.js [hunk 1]
```diff
@@ -20,17 +20,32 @@ export default Component.extend({
...
     if(isEmpty(events)){
       return [];
     }
     return events.filter(
       event => moment(event.startDate).isSame(moment(event.endDate), 'day')
     );
   }),
-  multiDayEventsList: computed('calendarEvents.[]', function(){
-    const events = this.get('calendarEvents');
+  multiDayEventsList: computed('nonIlmPreWorkEvents.[]', function(){
```
Comment (stopfstedt) [inline addon/components/ilios-calendar-week.js]: i see you shortened this to `multiDayEvents` elsewhere. is this an omission here, or legit?

File: addon/templates/components/ilios-calendar-week.hbs [hunk 2]
```diff
@@ -54,6 +54,8 @@
   </div>
 {{/el-weekly-calendar}}
 
+<IliosCalendarPreWorkEvents @events={{ilmPreWorkEvents}} />
+
 {{ilios-calendar-multiday-events
   events=multiDayEventsList
```
Comment (stopfstedt) [inline addon/templates/components/ilios-calendar-week.hbs]: see comment re: shortened CP name in component above.

File: addon/components/ilios-calendar-week.js (hunk 1 above)
Comment (stopfstedt) [reply addon/components/ilios-calendar-week.js]: nvm

**Pull request merged**